Samples/WMCConsole_winrtcpp/WMCConsole_winrtcpp.cpp

```diff
@@ -0,0 +1,255 @@
+// 
+// WMCConsole_winrtcpp.cpp : This file contains the 'main' function. Program execution begins and ends there.

```

Comment:
add copyrights

Reply:
Do we add copyrights to samples that we want people to copy...